database/migrations/2023_06_07_000001_create_pulse_tables.php

```diff
@@ -23,6 +23,7 @@ public function up(): void
             match ($this->driver()) {
                 'mysql' => $table->char('key_hash', 16)->charset('binary')->virtualAs('unhex(md5(`key`))'),
                 'pgsql' => $table->uuid('key_hash')->storedAs('md5("key")::uuid'),
+                'sqlite' => $table->string('key_hash'),
```

Comment:
SQLite doesn't have many data types so I think a varchar is as good as it gets. It also doesn't have any hashing functions built-in, so the hashing needs to be in PHP.